CHANGELOG.md

```diff
@@ -10,6 +10,20 @@ _None_
 
 ### New Features
 
+_None_
+
+### Bug Fixes
+
+_None_
+
+### Internal Changes
+
+_None_
+
+## 4.2.0
+
+### New Features
```

Comment:
Nit: this section contains both a new entry… and still kept the `_None_` line as well (that we were supposed to have removed when we added that first entry for #360 back then).

So you might as well take the occasion of this release PR to delete line 30 (sadly GitHub doesn't allow me to comment on line 30 to make a diff suggestion because that line is not part of this PR's diff 😝)

Reply:
Nice and easy – addressed by https://github.com/wordpress-mobile/release-toolkit/pull/368/commits/d1e176aae92161376bb025f0659d29da37f1dbcf